### PR TITLE
Fix #314, automatically insert the new step, or popout the popovers so the user sees where to click next

### LINKED ISSUE #314: Create Integration: Add to Integration Buttons Nonfunctional
"Rest stop" displayed to users for the first time upon completion of configuring both connections displays buttons in the main content area that are nonfunctional (ie clicking them does not result in an action).

Design:

save or add step

### PR DESCRIPTION
**Pull request by gashcrumb**

### CODE REVIEW

**Comment by gashcrumb** [comment]
apply this one first -> https://github.com/redhat-ipaas/ipaas-ui/pull/323

File: src/app/integrations/edit-page/save-or-add-step/save-or-add-step.component.ts [hunk 1]
```diff
@@ -79,6 +79,20 @@ export class IntegrationsSaveOrAddStepComponent extends FlowPage implements OnIn
...
   }
 
+  addNew(type: string) {
+    this.currentFlow.events.emit({
+      kind: 'integration-add-step',
+      type: type,
+    });
+  }
+
+  showPopouts(type: string) {
```
Comment (kahboom) [inline src/app/integrations/edit-page/save-or-add-step/save-or-add-step.component.ts]: How are we handling _which_ popover to show? Or does it not matter because only one will ever be present? We need to test manually for any edge cases.
